app/models/ama_subscriber.rb

```diff
@@ -0,0 +1,44 @@
+# rubocop:disable Metrics/LineLength
+# == Schema Information
+#
+# Table name: ama_subscribers
+#
+#  id         :integer          not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  ama_id     :integer          not null, indexed, indexed => [user_id]
+#  user_id    :integer          not null, indexed => [ama_id], indexed
+#
+# Indexes
+#
+#  index_ama_subscribers_on_ama_id              (ama_id)
+#  index_ama_subscribers_on_ama_id_and_user_id  (ama_id,user_id) UNIQUE
+#  index_ama_subscribers_on_user_id             (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_3b1c2ec3ee  (user_id => users.id)
+#  fk_rails_4ac07cb7f6  (ama_id => amas.id)
+#
+# rubocop:enable Metrics/LineLength
+
+class AmaSubscriber < ApplicationRecord
+  belongs_to :user, required: true
+  belongs_to :ama, required: true, counter_cache: true
+
+  validates :ama_id, uniqueness: { scope: :user_id }
+
+  after_commit(on: :create) do
+    ama.posts.each do |post|
+      user.notifications.follow(post.comments_feed, scrollback: 0)
+    end
+    user.notifications.follow(ama.original_post.comments_feed, scrollback: 0)
```

Comment:
Then it seems like we'll need a background worker checking for ama's that are "about to start" ( == 1 hour before? ok) and then send out notifications to all the subscribers and not have them follow the original post comments feed.